all/policies/greedy.py

```diff
@@ -70,13 +70,13 @@ def __init__(
     def __call__(self, state):
         best_actions = torch.argmax(self.q(state), dim=-1)
         random_actions = torch.randint(0, self.n_actions, best_actions.shape, device=best_actions.device)
-        choices = (torch.randn_like(best_actions) < self.epsilon).int()
+        choices = (torch.rand(best_actions.shape, device=best_actions.device) < self.epsilon).int()
```

Comment:
Why not rand_like?

Reply:
No particular reason, I just thought it should be consistent.

Reply:
That is a fair point. Why don't we use rand_like and randint_like?

Reply:
So I changed the rand to rand_like. randint_like is a bit awkward to use, so I just left it.

Reply:
Ah shoot, this creates some random other issue. I guess just put it back the way you had it, sorry. I'll approve. You can squash & merge once the CI passes.